### PR TITLE
Compute correct transaction value for exchange/field bundles

### LINKED ISSUE #340: Incorrect transaction value for bundles created by exchanges
### Bug description
Bundles with multiple outputs leads to an incorrect calculation of transaction value. 

Related issue: https://github.com/iotaledger/trinity-wallet/issues/312 (This issue was related to only failed transactions)

### Environment Specs

- Manufacturer: Apple
- Model: 6
- OS version: 11.4
- Trinity version:  v(0.4.1) 17

### Steps To Reproduce

Pull in transactions for bundle `9RIIFXWDJAGSTXLVILRABQWIIWCQNIRSKTGCLCFQNJISXTEB9BFOKVYYEADWYBHGBLJWQJEUWAJZPKFTW` and notice transaction value

### Expected behaviour
It should calculate and display correct transaction value

### Actual behaviour
Displays incorrect transaction value

### Am I planning to do it myself with a PR?
Yes.

### PR DESCRIPTION
**Pull request by laumair**
# Description

Fixes https://github.com/iotaledger/trinity-wallet/issues/340

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

# How Has This Been Tested?

- Added unit test coverage

# Checklist:

_Please delete items that are not relevant._

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] For changes to `mobile` that include native code (including React Native modules): I have verified that both iOS and Android successfully build in both `Debug` and `Release` modes
- [ ] For changes to `shared`: If applicable, I have verified that my changes are implemented correctly in `desktop` and `mobile`

### CODE REVIEW

File: src/shared/__tests__/libs/iota/transfers.spec.js [hunk 1]
```diff
@@ -43,30 +43,127 @@ describe('libs: iota/transfers', () => {
...
+            });
+
+            describe('with no input addresses belong to user addresses', () => {
+                it('should return a sum of all user output addresses', () => {
+                    expect(
+                        getTransferValue(
+                            [
+                                {
+                                    value: -10,
+                                    currentIndex: 0,
```
Comment (rajivshah3) [inline src/shared/__tests__/libs/iota/transfers.spec.js:132]: Wouldn't the transaction with a `currentIndex` of 0 be an output/have a positive value?

Comment (laumair) [reply src/shared/__tests__/libs/iota/transfers.spec.js:132]: I don't think it's necessary but I'll sort the indexes. Good catch.

Comment (rajivshah3) [reply src/shared/__tests__/libs/iota/transfers.spec.js:132]: Yeah, I didn't think it would affect anything really

Comment (rajivshah3) [reply src/shared/__tests__/libs/iota/transfers.spec.js:132]: Actually this isn't a big deal, we can merge this anyway

**Review by rajivshah3 — APPROVED** [review]

**Pull request merged**